src/components/Form/Gemstone.vue

```diff
@@ -31,30 +31,6 @@
     required
   ></v-text-field>
 
-  <v-text-field
-    v-model="formData.dimensionX"
-    :counter="10"
-    :rules="numberFieldRules"
-    label="dimensionX"
-    required
-  ></v-text-field>
-
-  <v-text-field
-    v-model="formData.dimensionY"
-    :counter="10"
-    :rules="numberFieldRules"
-    label="dimensionY"
-    required
-  ></v-text-field>
-
-  <v-text-field
-    v-model="formData.dimensionZ"
-    :counter="10"
-    :rules="numberFieldRules"
-    label="dimensionZ"
-    required
-  ></v-text-field>
-
```

Comment:
In the creation form we need to provide the dimensions because the request still takes dimensions. We just need to remove them from the visualization in the table.

Reply:
Done